### ISSUE TITLE
Aerial interface from Pneumaticcraft

### ISSUE THREAD

**Issue by jilleJr** [OpenMods/OpenPeripheral-Integration#8]
The mod Pneumaticcraft by MineMaarten got this block called Aerial interface which works like a remote inventory to whoever placed it. You may have heard of it and its really cool.

But I can not access the inventory with openperipherals integration because its overridden by the pneumaticMachine peripheral (which contains methods to check its pressure levels and such) from pneumaticcraft.

Now I did make an issue over on pneumaticcrafts github here
https://github.com/MineMaarten/PneumaticCraft/issues/362
and MineMaarten tried to only assign the special pneumaticcraft peripheral on the top and bottom side of the Aerial interface, but that sadly didn't do it. I still can't access the players inventory with OpenPeripherals.

Would be great if you could look into it and maybe see if it's plausible? Maybe you can combine your peripheral with his..

**Comment by Vexatos** [comment]
@jilleJr Seeing what MineMaarten did, it should be possible to access OpenPeripheral functions when placing the computer on the side of the interface. Are you sure that doesn't work?

**Comment by jilleJr** [comment]
@Vexatos  Yes I am sure. Maybe it's some bug from his part because I can still access the pneumaticcraft peripheral from the sides...

**Comment by jilleJr** [comment]
Just found out the perfect temporary solution: [Factorization Parasieve](http://purpleposeidon.github.io/FzDocs/parasieve.html)
> Place the parasieve against an inventory. Any machine that attempts to access the parasieve will access that inventory through it.

This block does as expected and the peripheral for it works as expected as well. Just perfect with some more prefect on top.
I'm so happy because I can now play with my dreams and automate my own inventory. \**so cool*\*
